crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -650,6 +661,37 @@ pub async fn resolve(
             )
         }
         Request::Module { module, path } => {
+            // Warn if the package is known not to be supported by Turbopack at the moment.
+            if UNSUPPORTED_PACKAGES.contains(module) {
+                UnsupportedModuleIssue {
+                    context,
+                    package: module.into(),
+                    package_path: None,
+                }
+                .cell()
+                .as_issue()
+                .emit();
+            }
+
+            let package_path = path
+                .to_string()
+                .strip_prefix("\"/")
```

Comment:
The path is quoted? Is it guaranteed to be double quotes?

Reply:
Yeah, it looks like that's from Pattern's implementation for `Display`: https://github.com/vercel/turbo/blob/c01ad84a7ae65711031812666edd594a44c3c398/crates/turbopack-core/src/resolve/pattern.rs#L592

I'm going to see if there's a cleaner way of doing this.